e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js

```diff
@@ -3,15 +3,28 @@ const BRANCH_NAME = "main"; // Affects the `local` testing only. On CI is passed
 const BASE_ENV = {
   WATCH:
     process.env.SAMPLE_APP_ENVIRONMENT === "development" ? "true" : "false",
-  PREMIUM_EMBEDDING_TOKEN: process.env.CYPRESS_MB_ALL_FEATURES_TOKEN ?? "",
+  PREMIUM_EMBEDDING_TOKEN:
+    process.env.CYPRESS_MB_ALL_FEATURES_TOKEN ||
+    process.env.CYPRESS_ALL_FEATURES_TOKEN ||
+    process.env.MB_ALL_FEATURES_TOKEN ||
+    process.env.ENTERPRISE_TOKEN ||
```

Comment:
Curious why we take so many environment variables here, is it for compatibility?